package.json

```diff
@@ -23,7 +23,7 @@
     "rollback": "knex migrate:rollback",
     "rollback-one": "knex migrate:down",
     "start": "node dist/index.js",
-    "test": "set TEST_ENVIRONMENT=true && ts-mocha --require @babel/register --exit test/** test/**/*",
+    "test": "set TEST_ENVIRONMENT=true && ts-mocha --require @babel/register --exit test/** test/**/* frontend/src/test/**",
```

Comment:
not sure that we want to launch the frontend test from the backend ?

Reply:
I wasn't sure too. I really wanted to test it but didn't know how to do it properly since there is currently no test of this kind on frontend.
Do you want me to set up completely? (test on frontend package.json and add it to the CI pipeline?)